src/runtime/rpc/rpc_address.cpp

```diff
@@ -51,15 +54,15 @@ uint32_t rpc_address::ipv4_from_host(const char *name)
 
     addr.sin_family = AF_INET;
     if ((addr.sin_addr.s_addr = inet_addr(name)) == (unsigned int)(-1)) {
+        // TODO(yingchun): use getaddrinfo instead
         hostent *hp = ::gethostbyname(name);
-        int err = h_errno;
-
-        if (hp == nullptr) {
-            LOG_ERROR("gethostbyname failed, name = %s, err = %d.", name, err);
+        if (dsn_unlikely(hp == nullptr)) {
+            LOG_ERROR_F(
+                "gethostbyname failed, name = {}, err = {}", name, utils::safe_strerror(h_errno));
```

Comment:
Human-readable message for `::gethostbyname` can be got by `hstrerror(h_errno)`, however considering that soon we will use `getaddrinfo`, we can use `gai_strerror` instead then.